.ibm/pipelines/value_files/values_showcase.yaml

```diff
@@ -42,60 +42,57 @@ global:
-              frontend:
-                backstage.plugin-github-issues:
-                  mountPoints:
-                    - mountPoint: entity.page.issues/cards
-                      importName: GithubIssuesCard
-                      config:
-                        layout:
-                          gridColumn: "1 / -1"
-                        if:
-                          allOf:
-                            - hasAnnotation: github.com/project-slug
         - package: ./dynamic-plugins/dist/roadiehq-backstage-plugin-github-pull-requests
           disabled: false
-          pluginConfig:
-            dynamicPlugins:
-              frontend:
-                roadiehq.backstage-plugin-github-pull-requests:
-                  mountPoints:
-                    - mountPoint: entity.page.overview/cards
-                      importName: EntityGithubPullRequestsOverviewCard
-                      config:
-                        layout:
-                          gridColumnEnd:
-                            lg: "span 4"
-                            md: "span 6"
-                            xs: "span 12"
-                        if:
-                          allOf:
-                            - isGithubPullRequestsAvailable
-                    - mountPoint: entity.page.pull-requests/cards
-                      importName: EntityGithubPullRequestsContent
-                      config:
-                        layout:
-                          gridColumn: "1 / -1"
-                        if:
-                          allOf:
-                            - isGithubPullRequestsAvailable
         - package: ./dynamic-plugins/dist/backstage-plugin-github-actions
           disabled: false
+        - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-quay
+          disabled: false
+        - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-keycloak-backend-dynamic
+          disabled: false
+          pluginConfig:
+            catalog:
+              providers:
+                keycloakOrg:
+                  default:
+                    baseUrl: ${KEYCLOAK_URL}
+                    loginRealm: ${KEYCLOAK_REALM}
+                    realm: ${KEYCLOAK_REALM}
+                    clientId: ${KEYCLOAK_CLIENT_ID}
+                    clientSecret: ${KEYCLOAK_CLIENT_SECRET}
+        - package: ./dynamic-plugins/dist/backstage-plugin-kubernetes-backend-dynamic
+          disabled: false
           pluginConfig:
-            dynamicPlugins:
-              frontend:
-                backstage.plugin-github-actions:
-                  mountPoints:
-                    - mountPoint: entity.page.ci/cards
-                      importName: EntityGithubActionsContent
-                      config:
-                        layout:
-                          gridColumn: "1 / -1"
-                        if:
-                          allOf:
-                            - isGithubActionsAvailable
+            kubernetes:
```

Comment:
Why do you add the `pluginConfig` field here ? Do you want to explicitly replace the default config provided in [the included `dynamic-plugins.default.yaml` file](https://github.com/janus-idp/backstage-showcase/blob/main/dynamic-plugins.default.yaml#L174) ?

Reply:
@davidfestal there is a additional config for topology code decorators
```
  - group: 'org.eclipse.che'
    apiVersion: 'v2'
    plural: 'checlusters'
```

Reply:
I see.